packages/a11y/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@kaizen/a11y",
+  "version": "0.0.2",
```

Comment:
interesting version to start with 😅

Reply:
😬 I was testing this in Unified Navigation and each time I published a change I needed a new version.

Reply:
It doesn't _super_ matter what version this package starts with, as Lerna will publish it as `1.1.0`

Reply:
not 0.1.0? (not that I have any issues with that)